transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -153,7 +150,10 @@ public ChannelHandlerContext fireUserEventTriggered(Object event) {
     @Override
     public ChannelHandlerContext fireChannelRead(Object msg) {
         AbstractChannelHandlerContext next = findContextInbound();
-        ReferenceCountUtil.touch(msg, next);
+        if (ResourceLeakDetector.ENABLED) {
```

Comment:
@Scottmitch for comparsion, it's twice the improvement of what "io.netty.buffer.bytebuf.checkAccessible"  https://github.com/netty/netty/commit/1103379e0292ea3cedac655a9fd13395e111d1cd brings to this app when enabled (disabled on the screenshot)

it's still a small % of overall thing because read/write syscalls but perf is perf :smile: